h2o-core/src/main/java/water/fvec/Frame.java

```diff
@@ -1595,6 +1597,7 @@ public CSVStreamParams setSeparator(byte separator) {
     Chunk[] _curChks;
     int _lastChkIdx;
     public volatile int _curChkIdx; // used only for progress reporting
+    private final Map<Integer, String[]> escapedCategoricalLevels = new HashMap<>();
```

Comment:
Question: Is this being serialized anywhere? Can we make it transient?

Reply:
Went through usages, doesn't seem like it. `ExportTaskDriver` uses it in `compute2` method, locally. Not serializing it. The same applies to `FrameUtils.EstimateTaskSize`, which uses this as a local instance in `map` method. This is safe to be marked as transient.

Reply:
thank you, please make it transient then

Reply:
Done. Thank you.